tests/remote_point_evaluation/remote_point_evaluation_03.cc

```diff
@@ -0,0 +1,461 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2021 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+// Like remote_point_evaluation_01.cc but normal and curvature vector living on
+// background mesh.
```

Comment:
How is this different from `tests/remote_point_evaluation/remote_point_evaluation_02.cc`?

Reply:
I have updated the comment. This was just a copy&paste error.